descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,22 @@
   "id": "@project.artifactId@-@project.version@",
   "name": "Camnuda BPM Module",
   "provides": [
+    {
+      "id": "camunda-history",
```

Comment:
Does Kafka require this or is this a new feature being added?

Reply:
This is required for MIS Catalog reports and should be exposed.

Reply:
Sure. And I will include that issue in the staging to main PR for this sprint effort.